Sources/HaCWebsiteLib/Events/Event.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+protocol Event: PostCardRepresentable {
+    var title : String {get}
+    var time : DateInterval {get}
+    var tagLine : String {get}
+    var color : String {get}
+    var hypePeriod : DateInterval {get}
+    var tags : [String] {get}
+    var imageURL : String? {get}
+    var location : Location? {get} //for room names etc.
```

Comment:
I can't seem to find this `Location` type anywhere in Foundation or our repo. Could you tell me where this is coming from?

Reply:
I think it's just in the HaCWebsiteLib directory

Reply:
GitHub was weird and I missed the file, let's carry on.

Is the comment still necessary here?

Reply:
nope it's not necessary  -will fix now